CMakeLists.txt

```diff
@@ -451,7 +469,7 @@ if(NOT LATESTLAPACK_FOUND)
   set(ALL_TARGETS ${ALL_TARGETS} ${LAPACKLIB})
 endif()
 
-if(BUILD_TESTING OR LAPACKE_WITH_TMG)
+if(BUILD_TESTING)
```

Comment:
Travis shows an error. See https://travis-ci.org/github/Reference-LAPACK/lapack/builds/766858282
```
CMake Error in CMakeLists.txt:

  given target "tmglib" more than once.
```

I think the issue can be solved by just removing these 3 lines from the `CMakeLists.txt`.

Reply:
May I commit directly to your branch?

Reply:
Yes sorry! I was (and still am) very busy these past days so I didn't get around to it.
It's still in my to-do list, but I had to reduce the priority.
Feel free to commit, I have added you as a collaborator.

Reply:
Ah ok. Thanks! I just asked to check if my suggestion makes sense. Normally, I have the permission to modify the PRs directly 😊

Reply:
Thanks for making that change! Looks like all tests are passing